src/main/scala/akka/persistence/cassandra/journal/PubSubThrottler.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.persistence.cassandra.journal
+
+import scala.concurrent.duration.FiniteDuration
+
+import akka.actor.{ Actor, ActorRef }
+
+/**
+ * Proxies messages to another actor, only allowing identical messages through once every [interval].
+ */
+class PubSubThrottler(delegate: ActorRef, interval: FiniteDuration) extends Actor {
+  import PubSubThrottler._
+  import context.dispatcher
+
+  val intervalMs = interval.toMillis
+  var seen = Map.empty[Any, Long]
+  context.system.scheduler.schedule(interval, interval, self, Cleanup)
```

Comment:
OK, will do. 

Has someone thought about making timers "owned" by their target `ActorRef`? Seems like timers that you set up to an `ActorRef` should always stop when the target dies. Can't/shouldn't akka stop the timer automatically by `context.watch`-ing it?